tests/export_back_compat_test.py

```diff
@@ -358,15 +364,18 @@ def test_cpu_qr_lapack_geqrf(self, dtype_name="f32"):
       dict(testcase_name=f"_dtype={dtype_name}_{batched}",
            dtype_name=dtype_name, batched=batched)
       for dtype_name in ("f32",)
-      # For batched qr we use cublas_geqrf_batched
+      # For batched qr we use cublas_geqrf_batched/hipblas_geqrf_batched.
       for batched in ("batched", "unbatched"))
-  def test_cuda_qr_cusolver_geqrf(self, dtype_name="f32", batched="unbatched"):
+  def test_gpu_qr_solver_geqrf(self, dtype_name="f32", batched="unbatched"):
+    if jtu.test_device_matches(["cuda"]):
+      data = self.load_testdata(cuda_qr_cusolver_geqrf.data_2023_03_18[batched])
+    elif jtu.test_device_matches(["rocm"]):
+      data = self.load_testdata(rocm_qr_hipsolver_geqrf.data_2024_08_05[batched])
```

Comment:
won't this fail when run on another platform? I think you can add `else: self.skipTest`

Reply:
same above

Reply:
Done! Thanks!